src/CanvasRenderingContext2d.cc

```diff
@@ -4,7 +4,7 @@
 
 #include <algorithm>
 #include "backend/ImageBackend.h"
-#include <cairo/cairo-pdf.h>
+#include <cairo-pdf.h>
```

Comment:
Thanks for the fixes!

#1594 just fixed the JPEG guards. Could you rebase on master please to isolate this change?